src/AppLayout.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import Header from './Header'
+import Footer from './Footer'
+
+export default class AppLayout extends React.Component {
```

Comment:
Hat das einen bestimmten Grund, dass du in dieser Komponente als einziges class Syntax benutzt und nicht function?